src/assets/translations/en/main.json

```diff
@@ -676,6 +677,9 @@
     "approvingAsset": "Approving %{symbol}",
     "viewTransaction": "View Transaction",
     "unlimited": "Unlimited",
+    "streamStatus": "Stream Status",
+    "fetchingStreamingSwapData": "Loading...",
```

Comment:
Do we want to improve this copy? Thinking something along the lines of "Initiating swap..."  or "Trade started..."

Reply:
I opted to remove the copy because its both "loading streaming swap progress" and "unable to load streaming swap metadata". This and the other issues with contradicting statuses will be address in a follow up when we upgrade the thorchain swapper to provide accurate trade status